spinta/backends/postgresql/types/geometry/encode.py

```diff
@@ -42,6 +44,14 @@ def prepare_dtype_for_response(
 ):
     shape = to_shape(value)
     point = shape.centroid
-    params = urlencode({'mlat': point.x, 'mlon': point.y})
-    link = f'https://www.openstreetmap.org/?{params}#map=19/{point.x}/{point.y}'
-    return Cell(shape.wkt, link=link)
+
+    if WKBElement.srid in [4326, -1]:
+        x, y = point.x, point.y
+    else:
+        proj = Transformer.from_proj(WKBElement.srid, 4326, always_xy=True)
+        x, y = proj.transform(point.x, point.y)
```

Comment:
Please add tests, where this transformation is checked. Test should check at least two cases, when property is in WGS and in LKS. In one case (WGS) transformation is not needed, in other case transformation is needed. Also test this with different shapes, at least with point and polygon.

https://shapely.readthedocs.io/en/stable/manual.html#shapely.ops.transform